Kitodo-DataManagement/src/main/java/org/kitodo/data/database/enums/PreviewHoverMode.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.data.database.enums;
+
+public enum PreviewHoverMode {
+
```

Comment:
Could you add some short description comment to the individual options in this enum?

Reply:
Now documented.